locale/ar.po

```diff
@@ -0,0 +1,37 @@
+# Arabic translations for hack-extension package.
+# Copyright (C) 2020 THE hack-extension'S COPYRIGHT HOLDER
+# This file is distributed under the same license as the hack-extension package.
+# Automatically generated, 2020.
+#
+msgid ""
+msgstr ""
+"Project-Id-Version: hack-extension\n"
+"Report-Msgid-Bugs-To: \n"
+"POT-Creation-Date: 2020-04-20 11:13+0200\n"
+"PO-Revision-Date: 2020-04-20 11:13+0200\n"
+"Last-Translator: Automatically generated\n"
+"Language-Team: none\n"
+"Language: ar\n"
+"MIME-Version: 1.0\n"
+"Content-Type: text/plain; charset=ASCII\n"
```

Comment:
I'd guess this is the issue. It's not ASCII, it's UTF-8.

Reply:
Thanks Will. Looks like there are more issues.